thattr.h

```diff
@@ -117,13 +117,12 @@ struct thattr_obj {
   size_t m_id = 0,  ///< Internal ID.
     m_tree_level;  ///< Tree level of object, if table has tree structure.
   const char * m_tree_node_id;  ///< Tree node id.
-  void * m_data;  ///< User defined data.
```

Comment:
Aj toto by som potreboval zanechat, thattr je univerzalna tabulka pouzivana na exporty a kadeco ide tusim, to ze som m_data nepouzil, neznamena, ze ho nepouzijem v buducnosti.

Reply:
Tenhle member se používá s pointery typu `thdataobject`, můžu změnit jeho typ z `void*` na `thdataobject*`?